plugins/rapid7_vulndb/komand_rapid7_vulndb/util/util.py

```diff
@@ -0,0 +1,38 @@
+from insightconnect_plugin_runtime.exceptions import PluginException
+
+import random
+from functools import wraps
+import time
+from _datetime import datetime
+
+
+class Util:
+    def retry(tries: int, timeout: int, exceptions, backoff_seconds: int = 1):
+        """
+        @param tries: Amount of times function is to be retried
```

Comment:
Mike told me to use this [docstring style](https://thomas-cokelaer.info/tutorials/sphinx/docstring_python.html)

Reply:
Linked docstring style will be used in future updates.